pkg/config/config.go

```diff
@@ -340,7 +340,7 @@ func SetEnabledIdlePower(enabled bool) {
 // SetEnabledGPU enables the exposure of gpu metrics
 func SetEnabledGPU(enabled bool) {
 	// set to true if any config source set it to true
-	instance.Kepler.EnabledGPU = enabled || instance.Kepler.EnabledGPU
```

Comment:
if the original `instance.Kepler.EnabledGPU` is true, which seems the function will not set the GPU enable from true to false.